pkg/oc/cli/cmd/set/triggers.go

```diff
@@ -435,7 +434,29 @@ func (o *TriggersOptions) updateTriggers(triggers *TriggerDefinition) {
 		if o.FromBitbucket != nil && *o.FromBitbucket {
 			triggers.BitbucketWebHooks = nil
 		}
-		return
+		if o.Remove {
```

Comment:
You're already inside `if o.Remove` so this doesn't make sense. And if so the below condition `f o.Remove || !o.Append && len(o.FromImage) > 0 {` doesn't make sense either. Maybe you could refactor this bit to a helper function and make this a bit more readable?

Reply:
this was a leftover ;-)